### DISCUSSION TITLE
No event is fired when a release is updated

### DISCUSSION THREAD

**Discussion by Polaris-Dust** [community/community#24838] [Apps, API and Webhooks]
<p>Currently only  <strong>published</strong> events are fired.</p>
<p>A release update, like changing the type from <strong>prerelease</strong> to <strong>release</strong> for instance, doesn’t fire anything preventing us to rely on it to trigger workflows with Github actions.</p>

**Comment by Polaris-Dust** [comment]
<p>Any news on when/if release update events will be fired?</p>

**Comment by leocourson** [comment, marked as answer]
<p>New actions for releaseEvent have been added : <a href="https://developer.github.com/changes/" rel="nofollow noopener">https://developer.github.com/changes/</a>, but not for prerelease -&gt; release (only for release -&gt; prerelease).<br>
See message : <br>
<a href="https://github.community/t/webhook-releaseevent-prerelease-release-notification/13957">https://github.community/t5/GitHub-API-Development-and/Webhook-ReleaseEvent-prerelease-gt-release-notification/m-p/22612?advanced=false&amp;collapse_discussion=true&amp;q=prerelease&amp;search_type=thread</a></p>

**Comment by philfi** [comment]
<p>Looks like this has since been updated to support a <code>released</code> action when updating from prerelease -&gt; released!</p>
